src/dataSources/canvas/canvas-api.js

```diff
@@ -0,0 +1,25 @@
+const {RESTDataSource} = require('apollo-datasource-rest');
+
+class CanvasAPI extends RESTDataSource {
+    willSendRequest(request) {
+        console.log(this.context.Authorization)
```

Comment:
This log is probably here for debugging purposes, but please remove before merging